pages/_app.js

```diff
@@ -19,9 +19,7 @@ export default function MyApp({ Component, pageProps }) {
   useEffect(() => {
     if (!theme) return
     const $html = document.querySelector('html')
-    $html.classList.remove('light')
-    $html.classList.remove('dark')
-    $html.classList.remove('dim')
+    $html.classList.remove(...$html.classList)
     $html.classList.add(theme.toString())
```

Comment:
`$html.setAttribute('class', theme)`  Bu şekilde var olan class'ları remove etmeye gerek kalmadan tek satırda class'ı tekrar tanımlayabiliriz.

```js
useEffect(() => {
    if (!theme) return
    const $html = document.querySelector('html')
    $html.setAttribute('class', theme)
  }, [theme])
```

Reply:
bu şekilde bütün sınıfları silersin ve doğru olmaz. başka bir eklenti veya polyfill için sınıf ataması olmuş olabilir.